sdk/metric/meter.go

```diff
@@ -437,12 +440,14 @@ func (r observer) ObserveFloat64(o metric.Float64Observable, v float64, opts ...
 	}
 
 	if _, registered := r.float64[oImpl.observablID]; !registered {
```

Comment:
```suggestion
	if _, registered := r.float64[oImpl.observablID]; !registered && !oImpl.dropAggregation {
```

Reduces indentation.